lib/dom_utils.coffee

```diff
@@ -413,8 +413,9 @@ DomUtils =
     if not node?
       return null
     if node == sel.anchorNode and sel.focusOffset == sel.anchorOffset
-      # The selection either *is* an element, or is inside an opaque element (eg. <input>).
-      node = node.childNodes[sel.focusOffset]
+      # If the selection is not a caret inside a `#text`, which has no child nodes,
+      # then it either *is* an element, or is inside an opaque element (eg. <input>).
+      node = node.childNodes[sel.focusOffset] || node
```

Comment:
Why is this change needed?

Reply:
I found that a first <kbd>Escape</kbd> on HUD's input box can not exit FindMode on Chrome 74, if only the input box is empty, while it just blurs input. Only a second Escape exits it and make HUD hide.
So I add this check in the second commit of this PR to fix the crash behind this problem.